react/src/lib/Lightbox/index.js

```diff
@@ -556,6 +556,8 @@ Lightbox.propTypes = {
   onDownload: PropTypes.func,
   /** @prop Array of Lightbox pages */
   pages: PropTypes.array.isRequired,
+  /** @prop tooltip style */
```

Comment:
This line is just missing a pipe(|) with the default props like tooltips below. This is used for the docs site.

Reply:
@bfbiggs fixed